Lib/fontTools/feaLib/parser.py

```diff
@@ -663,6 +663,30 @@ def parse_subtable_(self):
         self.expect_symbol_(";")
         return ast.SubtableStatement(location)
 
+    def parse_size_parameters_(self):
+        assert self.is_cur_keyword_("parameters")
+        location = self.cur_token_location_
+        DesignSize = self.expect_decipoint_()
+        SubfamilyID = 0
+        RangeStart = 0
+        RangeEnd = 0
+        if self.next_token_type_ is Lexer.NUMBER:
```

Comment:
I think there's still a difference between your current code's behaviour and that of makeotf.
If I change the "feaLib/testdata/size2.fea" test case to only set the first parameter (DesignSize), like that:

```diff
feature size {
+  parameters 10.0 0;
-  parameters 10.0;
} size;
```

then feaLib sets the other three values (SubfamilyID, RangeStart and RangeEnd) to 0, whereas makeotf raises a syntax error:
```
syntax error at ";" missing { FONTREV NUM } [features 30]
```

Similarly, if I set `parameters 10.0 0 0;`, without the last RangeEnd value, makeotf raises another fatal error, which makes clear what it's expecting:

```
makeotfexe [FATAL] <Test-Regular> size feature must have 4 or 2 parameters if sub family code is zero!
```

Do you mind changing this to match makeotf?
Thanks